stingray/stats.py

```diff
@@ -337,6 +546,43 @@ def pds_probability(level, ntrial=1, n_summed_spectra=1, n_rebin=1):
     return epsilon
 
 
+def pds_logprobability(level, ntrial=1, n_summed_spectra=1, n_rebin=1):
```

Comment:
I am curious why this is new here. I thought we had a function for the PDF log-probability in this module before? What has changed?

Reply:
We had no logprobability, only probability